src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Car Registration</title>
+</head>
+<body>
+<h1>Please fill the car registration form below</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Car model <input type="text" name="model"><br>

```

Comment:
```suggestion
    Car model <input type="text" name="model" required><br>

```

Reply:
"required" specifies that an input field must be filled out before submitting the form. Same with the Driver and Manufacturer